provelo_customization/report/available_holidays_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.actions.act_window" id="hr_holidays.action_hr_available_holidays_report">
+        <field name="name">Leaves Analysis</field>
+        <field name="res_model">hr.holidays</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">graph,pivot,calendar,form</field>
+        <field name="view_id"></field>  <!-- force empty -->
```

Comment:
Why do you need to do this?

Reply:
It's from a copy/paste the source code and added an additional `context` key-value.
Source code : 
```xml
        <record model="ir.actions.act_window" id="action_hr_available_holidays_report">
            <field name="name">Leaves Analysis</field>
            <field name="res_model">hr.holidays</field>
            <field name="view_type">form</field>
            <field name="view_mode">graph,pivot,calendar,form</field>
            <field name="view_id"></field>  <!-- force empty -->
            <field name="context">{'search_default_leaves':1}</field>
            <field name="search_view_id" ref="view_hr_holidays_filter"/>
        </record>
```

Reply:
changed in last commit